kork-sql/src/main/kotlin/com/netflix/spinnaker/kork/sql/config/DefaultSqlConfiguration.kt

```diff
@@ -72,6 +73,11 @@ class DefaultSqlConfiguration {
   fun liquibase(properties: SqlProperties): SpringLiquibase =
     SpringLiquibaseProxy(properties.migration)
 
+  @Bean
+  @ConditionalOnProperty("sql.secondaryMigration.jdbcUrl")
```

Comment:
needs to be kebab-case for boot2:
```suggestion
  @ConditionalOnProperty("sql.secondary-migration.jdbc-url")
```